src/helpers/copy-to-clipboard.js

```diff
@@ -1,3 +1,9 @@
 export default function copyToClipboard(content) {
-    //ma kopiować do schowka
+    navigator.clipboard.writeText(this.coloredCode)
```

Comment:
Co to jest this.coloredCode? Jesteś w innym pliku. Nie ma tu this z komponentu

Reply:
U mnie to wygląda tak że chciałem tak zrobić żeby bylo writeText(this.code lub this.coloredCode) ktora jest przez slowo kluczowe this odwolane do zasięgu wyżej i w tym przypadku jeśli eksportujemy funkcję to idzie do komponentu code.vue gdzie z kolei jest wpisana funkcja coloredCode() która zwraca this.code i jest to powiązane z code=„<h1>test</h1>”. Brzmi szalenie ?